### PR TITLE
update the authority class name 

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

**Review by huihuiw01 — APPROVED** [review]

**Review by Xiaochao8 — APPROVED** [review]

File: g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/SwaggerConfig.java [hunk 1]
```diff
@@ -89,7 +89,7 @@
 	@ConditionalOnProperty(value = "vipservice.authority.enable")
```
Comment (Xiaochao8) [inline g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/SwaggerConfig.java:89]: Does here need to change?

File: g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/login/controller/AuthenticationLoginController.java [hunk 2]
```diff
@@ -32,10 +31,10 @@
 
 
 
-@RestController("Authorization-authenticationLoginAPI")  
+@RestController("Authentication-authenticationLoginAPI")
```
Comment (Xiaochao8) [inline g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/login/controller/AuthenticationLoginController.java:34]: Does it need to change file name from authorization to authentication?

**Review by Xiaochao8 — CHANGES_REQUESTED** [review]

File: g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/SwaggerConfig.java (hunk 1 above)
Comment (tigershi) [reply g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/SwaggerConfig.java:89]: this is not need to change

File: g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/login/controller/AuthenticationLoginController.java (hunk 2 above)
Comment (tigershi) [reply g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/login/controller/AuthenticationLoginController.java:34]: Yes, I have confirm it with Nannan

**Review by Xiaochao8 — APPROVED** [review]

**Pull request merged**